app/assets/stylesheets/pages/_index.scss

```diff
@@ -1,2 +1,9 @@
 // Import page-specific CSS files here.
 @import "home";
+
+body {
+  height: 100vh;
```

Comment:
use min-height instead of height, or it will have a fix height of 100vh and won't be able to expand if the content inside it passes the viewport height. There is also a way to make it occupy the whole viewport without depending on the vh, that is a rather unstable unit.

:point_down: :point_down: check this out :point_down: :point_down: 
https://stackoverflow.com/questions/17555682/height-100-or-min-height-100-for-html-and-body-elements